assets/js/collaborative-editor/components/AdaptorDisplay.tsx

```diff
@@ -72,9 +78,19 @@ export function AdaptorDisplay({
   // Safe package name for icon (fallback to empty string)
   const safeAdaptorPackage = adaptorPackage || '';
 
+  // Get credential from store if we have an ID
+  const credential = credentialId ? findCredentialById(credentialId) : null;
+
   // Check if credential is connected
   const hasCredential = !!credentialId;
 
+  // TODO - come back to pulse concept later
+  // // Check if adaptor is language-common (shouldn't pulse for common)
+  // const isLanguageCommon = adaptorPackage === '@openfn/language-common';
+
+  // // Should pulse when: no credential AND not language-common
+  // const shouldPulse = !hasCredential && !isLanguageCommon;
```

Comment:
Should we remove this ? Maybe have an issue or implement now. But I don't think we should leave it here